src/app/domain/case-viewer/components/summary-panel/summary-panel.component.html

```diff
@@ -1,7 +1,29 @@
 <div data-selector="summary-panel-component" class="govuk-grid-row">
-    <ng-container *ngFor="let section of panelData.sections">
-        <div class="govuk-grid-column-one-half">
-            <app-data-list [title]="section.name" [dataList]="section.fields"></app-data-list>
-        </div>
-    </ng-container>
+    <div class="govuk-grid-column-two-thirds">
+        <ng-container *ngFor="let section of panelData.sections">
+            <ng-container [ngSwitch]="section.type" >
+                <app-data-list
+                    data-selector="data-list"
+                    *ngSwitchCase="'data-list'"
```

Comment:
We shouldn't need a switch here if everything here is a datalist

Reply:
everything isn't all data-list timeline is also one of these